### DISCUSSION TITLE
Issue with Empty Folders when they actually contain data - AWS S3

### DISCUSSION THREAD

**Discussion by wilmaldonado** [iterate-ch/cyberduck#15436] [Q&A]
•	Problem: Folders occasionally appear empty when they should contain files.
•	Resolution Attempts:
        1.  Right-click on the folder and select "Reload" in the MD menu (50% success rate).
         2. Shut down MD and restart it (75% success rate).

**Comment by tatze96** [comment]
Given that you are using the *Smart Synchronization* mode, this issue might be caused by paused synchronization.

The synchronization process can be paused manually but also gets paused by Mountain Duck if the connection to the server is lost or too unstable to transfer files. Files and folders that weren't indexed before the server or network connection dropped won't show up if you work offline. Mountain Duck will resume the synchronization automatically once the connection can be established again in case it was paused automatically.

Please verify that the synchronization process isn't paused within the Mountain Duck menu if the file listing is empty again.
If the synchronization isn't paused, send us the logging output with enabled [debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log) right after the file listing issue occurs again to support@mountainduck.io with the ticket reference GH-15436 for further investigation.

**Comment by wilmaldonado** [comment]
After moving from Smart Sync to Online mode none of my coworkers have reported any more problems with empty folders.
